platform/windows/display_server_windows.cpp

```diff
@@ -3522,7 +3520,8 @@ DisplayServer::WindowID DisplayServerWindows::_create_window(WindowMode p_mode,
 
 		if (is_dark_mode_supported()) {
 			BOOL value = is_dark_mode();
-			::DwmSetWindowAttribute(wd.hWnd, DWMWA_USE_IMMERSIVE_DARK_MODE, &value, sizeof(value));
+			::DwmSetWindowAttribute(wd.hWnd, immersive_dark_mode, &value, sizeof(value));
+			SetWindowPos(wd.hWnd, 0, 0, 0, 0, 0, SWP_DRAWFRAME | SWP_NOACTIVATE | SWP_NOMOVE | SWP_NOSIZE | SWP_NOZORDER);
```

Comment:
This line can also be removed because it has no effect. The window title bar doesn't draw correctly with or without this line included.